www/templates/html/Controller.tpl.php

```diff
@@ -1,27 +1,32 @@
 <?php
-require_once 'UNL/Templates.php';
+use UNL\Templates\Templates;
 
-UNL_Templates::$options['version'] = 4.0;
+$page = Templates::factory('Fixed', Templates::VERSION_4_1);
 
-$template = 'Fixed';
 $baseUrl = UNL_MediaHub_Controller::getURL();
 
-$page = UNL_Templates::factory($template);
-
-if (isset($GLOBALS['UNLTEMPLATEDEPENDENTSPATH'])) {
```

Comment:
The idea being to not require the presence of the directory (say if you are in a vagrant box)